source/matplot/core/axis_type.cpp

```diff
@@ -247,9 +252,17 @@ namespace matplot {
                 // So if there is no explicit label but there is
                 // a format we want to apply, we also need to explicitly
                 // create a label according to our format.
-                r += "\"" +
-                     escape(num2str(tick_values_[i], tick_label_format_)) +
-                     "\" ";
+                if (is_timestamp_) {
+                    char buff[20] = {0};
+                    time_t now = (time_t)tick_values_[i];
+                    strftime(buff, 20, tick_label_format_.c_str(),
```

Comment:
Will the default `tick_label_format_` work for timestamps?

Reply:
No, because `strftime` supports formatting the hour, minute, second, etc using a millisecond input. `num2str` does not do this